core/src/main/java/cucumber/runtime/RuntimeOptions.java

```diff
@@ -56,6 +56,7 @@ public String map(String keyword) {
             return keyword.replaceAll("[\\s',!]", "");
         }
     };
+    private static boolean wip = false;
```

Comment:
Why static state? Please change to instance state.

Reply:
> Need code review, especialy for the use of static attribute for wip.

I have do that because it is dificult to access RuntimeOption instance in Result.

I will find anthor way to access it.